spec/services/datetime_parser_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DatetimeParser do
+  describe "#parse" do
+    it "returns the parsed date/time when valid" do
```

Comment:
There are a lot of things being tested here, at it's not immediately clear what being "valid" means. Can you split this test up into smaller units that describe what's valid about the date/time.  e.g.
"it accepts a time that contains a pm"

Reply:
Agreed 👍 was copied directly from content publisher but happy to improve it!